network-api/networkapi/donate/templates/pages/landing_page.html

```diff
@@ -0,0 +1 @@
+<h1> {{ page.title }} </h1>
```

Comment:
Since the ask of this ticket was just to handle the backend work of creating the site / model, this template just renders the page title. The front end ticket is #10266